src/databricks/labs/ucx/hive_metastore/table_migrate.py

```diff
@@ -489,3 +491,118 @@ def _sql_alter_from(self, table: Table, target_table_key: str, ws_id: int):
             f"('upgraded_from' = '{source}'"
             f" , '{table.UPGRADED_FROM_WS_PARAM}' = '{ws_id}');"
         )
+
+
+class ACLMigrator:
+    def __init__(
+        self,
+        table_crawler: TablesCrawler,
+        grant_crawler: GrantsCrawler,
+        ws: WorkspaceClient,
+        workspace_info: WorkspaceInfo,
+        backend: SqlBackend,
+        group_manager: GroupManager,
+        migration_status_refresher: MigrationStatusRefresher,
+        principal_grants: PrincipalACL,
+    ):
+        self._tc = table_crawler
+        self._gc = grant_crawler
+        self._backend = backend
+        self._workspace_info = workspace_info
+        self._workspace_name = workspace_info.current()
+        self._ws = ws
+        self._group = group_manager
+        self._migration_status_refresher = migration_status_refresher
+        self._seen_tables: dict[str, str] = {}
+        self._principal_grants = principal_grants
+
+    def migrate_acls(
+        self, target_catalog: str | None = None, legacy_table_acl: bool = True, principal: bool = True
```

Comment:
```suggestion
        self, target_catalog: str | None = None, legacy_table_acl: bool = True, principal: bool = True,
```
so that formatting is better

Reply:
Fixed